test/models/idea_test.exs

```diff
@@ -17,6 +17,28 @@ defmodule RemoteRetro.IdeaTest do
     assert user_id_error == "can't be blank"
   end
 
+  describe "validating assignee_id presence" do
+    test "assignee_id required when idea is action-item" do
+      changeset = Idea.changeset(%Idea{category: "action-item", assignee_id: nil})
+
+      {asignee_id_error, _} = Keyword.fetch!(changeset.errors, :assignee_id)
+
+      assert asignee_id_error == "can't be blank"
+    end
+
+    test "assignee_id and category action-item do not trigger error" do
+      changeset = Idea.changeset(%Idea{category: "action-item", assignee_id: 1})
+
+      refute Keyword.has_key?(changeset.errors, :assignee_id)
+    end
+
+    test "assignee_id NOT required when idea category NOT action-item" do
+      changeset = Idea.changeset(%Idea{category: "happy", assignee_id: 1})
```

Comment:
i think we'll want the `assignee_id` to be `nil` here! change that and we're golden!